packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -233,6 +239,7 @@ class VideoEvent {
             eventType == other.eventType &&
             duration == other.duration &&
             size == other.size &&
+            rotationCorrection == other.rotationCorrection &&
```

Comment:
You are using "==" with irrational numbers.  This is error prone.

Reply:
Changed to int